docs/2.getting-started/2.getting-started.mdx

```diff
@@ -0,0 +1,58 @@
+---
+name: Getting Started in 5 Steps
+route: /getting-started/
+---
+
+import { NextButton } from '../components/nextButton.tsx'
+import { PrevButton } from '../components/prevButton.tsx'
+
+# Getting Started in 5 Steps
+
+Let's create our first Botonic bot!
+
+## Step 1 – Install the Botonic Command Line Interface
+1. Install the LTS v12 of [NodeJS installer](https://nodejs.org/). You can also install Yarn.
+2. Verify the installation version by running `npm --version` and `node --version`. **Note:** You must have at least `node >= 10.0.0` version installed.
```

Comment:
At the end of the sentence I will put it here `You can also use [Yarn](https://yarnpkg.com/).` as part of the note.